akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -151,11 +118,9 @@ object Sink extends SinkApply {
    * element is dropped and the stream continues.
    *
    * @see [[#mapAsyncUnordered]]
-   */
+   */ // FIXME Why is this method even useful?
```

Comment:
this is similar to foreach, whatever task foreach launched asynchronously in the background is out of scope from our shutdown.

In the case of a connection stream, if you terminate the bind-actor, then it terminates its accepted connections. If you just cancel the bind-stream, it will keep existing children around, and shut down the actor when all the children are gone. Unsubscribed children stop themselves after 5 seconds timeout, or when the parent actor stops.